monash-human-power/dashboard
# Add V3 structure and docs

## Description

This PR is very simple and does 3 main things:
- Adds folders for V2/v3/common components, API and views
- Updates documentation about folder structure
- Adds correct V3 routing

## Screenshots

### **NEW** V3 routing
![image](https://user-images.githubusercontent.com/23159604/106081722-8542a900-616d-11eb-883c-282b7e93b612.png)

### **NEW** Component documentation preview
![image](https://user-images.githubusercontent.com/23159604/106081368-e4ec8480-616c-11eb-86cb-77f6c9dbe70b.png)

### **NEW** V2 Component documentation preview
![image](https://user-images.githubusercontent.com/23159604/106081405-f3d33700-616c-11eb-81a8-6b2e2d323097.png)


## Steps to Test
1. Read the new documentation here:

[**NEW** API documentation](https://github.com/monash-human-power/dashboard/tree/blake-haydon/add-v3-structure/client/src/api/README.md)

[**NEW** components documentation](https://github.com/monash-human-power/dashboard/tree/blake-haydon/add-v3-structure/client/src/components/README.md)

2. Check that the routing works by running `yarn install && yarn start` while in the client folder (with the server running in the background)
